flask_appbuilder/models/sqla/interface.py

```diff
@@ -25,6 +25,10 @@ def _include_filters(obj):
         if not hasattr(obj, key):
             setattr(obj, key, getattr(filters, key))
 
+def _is_sa_type(obj, sa_type):
```

Comment:
[optional] `_is_sqla_type` is more clear than `_is_sa_type` to me

Reply:
sounds good